.github/workflows/publish-prod.yml

```diff
@@ -5,7 +5,10 @@
     "permissions":
       "contents": "read"
       "packages": "write"
-    "runs-on": "ubuntu-latest"
+      "pull-requests": "read"
+    "runs-on":
```

Comment:
This is running on our runners now, but I think this is ok for deploys, as they require approval?

Reply:
well spotted. that is not intentional
in fact in the past this didn't even work for some reason

i'll update it. it was meant to run on the gh runners

Reply:
this was an upstream regression.
i pulled the fix (it was already upstream)